pipelines/youtube_data/__init__.py

```diff
@@ -0,0 +1,181 @@
+import datetime
+from typing import Any, Dict, Iterator, List, Sequence
+
+from googleapiclient.discovery import build
+
+import dlt
+from dlt.extract.source import DltResource, DltSource
+from dlt.common.typing import TDataItem
+
+
+_SERVICE_NAME = "youtube"
+_SERVICE_VERSION = "v3"
+current_time = datetime.datetime.now()
+
+def _get_channel_id(youtube: Any, channel_name: str) -> List[str]:
+    """
+    function to retrieve channel id given channel custom name e.g. MrBeast
+
+    params:
```

Comment:
Same here. Also, the docstring layout looks like Google style docstrings. To be consistent with this style it's best to use `Args:` for naming function arguments.
See more examples here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html